src/d3a/constants.py

```diff
@@ -56,3 +56,12 @@
 CN_PROFILE_EXPANSION_DAYS = 7
 
 RUN_IN_REALTIME = False
+
+
+class SettlementTemplateStrategiesConstants:
```

Comment:
Potentially, this could be part of ConstSettings, right?

Reply:
That's what I was also in-between. Research / UI have not yet decided on the way that the settlement markets are configured for template strategies. If the settlement market rates can be configured by the UI, you are right that this should be in the ConstSettings. If this is not the case though, and the rates of the template strategies will only be configurable in d3a, then the right place is not the d3a-interface but the constants of d3a, that's why I placed them here. 
I can move it now, however I am not sure if at the very end it will be useful. If you believe that it is better to be future-proof and move this to d3a-interface please ping me and I'll do it, thanks!

Reply:
Convinced! Leave it please. Thanks!